addon/serializers/application.js

```diff
@@ -86,9 +86,18 @@ export default DS.RESTSerializer.extend({
         // the links property so the adapter can async call the
         // relationship.
         // The adapter findHasMany has been overridden to make use of this.
-        if ( options.relation ) {
-          hash.links = {};
-          hash.links[key] = { type: relationship.type, key: key };
+        if(options.relation) {
+          // hash[key] contains the response of Parse.com: eg {__type: Relation, className: MyParseClassName}
+          // this is an object that make ember-data fail, as it expects nothing or an array ids that represent the records
+          hash[key] = [];
+
+          // ember-data expects the link to be a string
+          // The adapter findHasMany will parse it
+          if (!hash.links) {
+            hash.links = {};
+          }
+
+          hash.links[key] = JSON.stringify({typeKey: relationship.type.typeKey, key: key});
```

Comment:
https://github.com/joshfester/ember-parse-adapter/commit/121909ddcc417f580b931387888c7ce5d75d722a introduced the JSON.stringify() call and https://github.com/joshfester/ember-parse-adapter/commit/337d2ff8fc4c3d6afeccf65145e71ca025623f99 changed "type" to "typeKey"

I mean to include the commit messages from each of these hashes into this commit listed under each referenced hash.  If this is preferred I can cancel this pull request and re-commit.